dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -1,19 +1,65 @@
+import { ArticleSpecial, ArticleDesign } from '@guardian/libs';
 import { decideFormat } from '../web/lib/decideFormat';
 
+/**
+ *
+ * This function makes the decision about when we use the parent's (the
+ * container's) format property to override the styling of a sublink, and
+ * when we allow the sublink to express its own styling.
+ *
+ * Eg. If you had a sublink to a lifestyle article, when should it use pink for
+ * the kicker and when would that not look right
+ *
+ * @returns the format property that we will use to style the sublink
+ */
+const decideSubLinkFormat = ({
+	linkFormat,
+	containerFormat,
+	containerPalette,
+}: {
+	linkFormat?: CAPIFormat;
```

Comment:
Might not be necessary for this PR, but should we add a front enhancer which coverts the `card.format` from `CAPIFormat` to `ArticleFormat`? We should avoid having two different format types floating around the rendering layers if possible